packages/mdc-textfield/_mixins.scss

```diff
@@ -486,24 +428,30 @@
   overflow: hidden;
 
   .mdc-text-field__input {
-    height: auto;
+    align-self: auto;
+    box-sizing: content-box;
+    height: 30px;
```

Comment:
I am confused by why this is changed

Reply:
Possibly related: this screenshot diff I took of part of the page that otherwise didn't change

(Also notice the label position changed)

![image](https://user-images.githubusercontent.com/398379/41178905-cc746a1e-6b36-11e8-90b7-c43e192accc9.png)

Reply:
Doesnt seem to show a difference in my comparison with master. Will follow up with you